### ISSUE TITLE
Table : When selected, entire row information is not conveyed by screen reader.

### ISSUE THREAD

**Issue by DikshaHub** [SAP/ui5-webcomponents#8264]
### Describe the bug

When we use a Voice Over software, such as JAWS on Windows and navigate to a row in the table, entire row information is not conveyed . Also it doesn't convey if any field inside the row is blank . This issue was raised by accessibility testing team. 

### Isolated Example

https://codesandbox.io/p/sandbox/charming-lena-78pr5k?file=%2Fsrc%2FApp.js%3A20%2C9

### Reproduction steps

1. Open the link.
2. Open JAWS software.
3. Select the first row and check the speech history.
4. Select the second row and check the speech history.
5. After selecting the second row, press TAB and check the speech history again.

### Expected Behaviour

1. When the first row is selected, it should reflect "From" and "To" in the speech history or audio. But it only reads "From" ("To" is skipped).
2. After selecting second row ,  entire row information should be conveyed by screen reader. But it doesn't read anything. 
3. Though when you press TAB and enter tablecell , it reads out that particular cell is blank. 

### Screenshots or Videos

<img width="714" alt="MicrosoftTeams-image (1)" src="https://github.com/SAP/ui5-webcomponents-react/assets/66899235/1cbd6552-6b58-4870-a4d3-12359658b75f">

### UI5 Web Components for React Version

~1.19.0

### UI5 Web Components Version

~1.17.0

### Browser

Chrome

### Operating System

Windows

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by yanaminkova** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,

Could you please have a look at this issue?
It is reproducible with the steps described above.

Thanks and KInd regards,
Yana

**Comment by elenastoyanovaa** [comment]
Hello @DikshaHub ,

I have tested with JAWS 2024 latest and Chrome latest on Windows 11 and VoiceOver on macOS and I am unable to reproduce the issue you are mentioning.
I am not sure what do you mean by selecting as the provided table has no selection mode, but only receives focus, so I have tested with TAB and focusing of a certain part of a table.

If you place the focus on the column header it will read out both column header texts + the text "Header Row". Then If you press arrow down in order to focus the row the speech output is as follows "From Days To Days". Both columns are spoken again + row count info. Then if one presses down again, the expected info will be read out for the last row as well.
Please make sure that the virtual cursor is OFF in order to escape JAWS reading mode, because by navigating in the page you are actually interacting with it so the application mode of JAWS should be turned on instead of the reading mode (JAWS key + Z is the combination to toggle between modes).

Regarding the statement that "When we use a Voice Over software, such as JAWS on Windows and navigate to a row in the table, entire row information is not conveyed. Also it doesn't convey if any field inside the row is blank"
If I understand clearly, when one focuses a row you expect to read something like this:
"From input blank days to input blank days row 1 of 3" which is currently limited to the column information + the textual content inside the cell and no other info regarding the cell contained items.
This is a known limitation and requires a specific implementation for the whole project. You can check more information here: https://github.com/SAP/ui5-webcomponents/issues/8022

Kind Regards,
Elena
